src/app/app.component.spec.ts

```diff
@@ -1,28 +1,13 @@
-import { TestBed } from '@angular/core/testing';
-import { RouterTestingModule } from '@angular/router/testing';
 import { AppComponent } from './app.component';
 
 describe('AppComponent', () => {
-  beforeEach(async () => {
-    await TestBed.configureTestingModule({
-      imports: [
-        RouterTestingModule
-      ],
-      declarations: [
-        AppComponent
-      ],
-    }).compileComponents();
-  });
+  let component: AppComponent;
```

Comment:
What about firebase-room-service, you racist!

Reply:
That service did not exist when I created the PR!

Reply:
Yeah right, I am gonna act like I believe you